samples/js/node/notebooks/213-question-answering.nnb

```diff
@@ -58,25 +58,21 @@
         {
             "language": "typescript",
             "source": [
-                "const core = new ov.Core();\n  const model = core.readModel(modelXMLPath);\n\n  new ov.PrePostProcessor(model)\n        .setInputElementType(0, ov.element.f32)\n        .build();\n  new ov.PrePostProcessor(model)\n        .setInputElementType(1, ov.element.f32)\n        .build();\n  new ov.PrePostProcessor(model)\n        .setInputElementType(2, ov.element.f32)\n        .build();\n  new ov.PrePostProcessor(model)\n        .setInputElementType(3, ov.element.f32)\n        .build();\n\n  const compiledModel = core.compileModel(model, 'CPU');\n\n  const inputs = compiledModel.inputs;\n  const outputs = compiledModel.outputs;\n\n  const inputSize = compiledModel.input(0).shape[1];"
+                "const core = new ov.Core();\nconst model = core.readModel(modelXMLPath);\n\nnew ov.PrePostProcessor(model)\n  .setInputElementType(0, ov.element.f32)\n  .setInputElementType(1, ov.element.f32)\n  .setInputElementType(2, ov.element.f32)\n  .setInputElementType(3, ov.element.f32)\n  .build();\n\nconst compiledModel = core.compileModel(model, 'CPU');\n\nconst inputs = compiledModel.inputs;\nconst outputs = compiledModel.outputs;\n\nconst inputSize = compiledModel.input(0).shape[1];"
             ],
             "outputs": []
         },
         {
             "language": "javascript",
             "source": [
-                "inputs.forEach(i => console.log(`${i}`));\nconsole.log('===')\noutputs.forEach(o => console.log(`${o}`));"
```

Comment:
I think we can list both inputs and outputs:
`inputs.forEach(i => console.log(`${i}`));`

Reply:
Added